src/qml/qgismobileapp.qml

```diff
@@ -3829,11 +3829,16 @@ ApplicationWindow {
   Changelog {
     id: changelogPopup
     objectName: 'changelogPopup'
-
     parent: Overlay.overlay
 
-    property var expireDate: new Date(2038, 1, 19)
-    visible: settings && settings.value("/QField/ChangelogVersion", "") !== appVersion && expireDate > new Date()
+    Component.onCompleted: {
+      const changelogVersion = settings.value("/QField/ChangelogVersion", "");
+      if (changelogVersion === "") {
+        settings.setValue("/QField/ChangelogVersion", appVersion);
+      } else if (changelogVersion !== appVersion) {
```

Comment:
I wonder if we should reduce this to minor updates only (and exclude patches), but this would need some more work to get there.